ideScripts/updateBuildData.py

```diff
@@ -134,8 +136,11 @@ def createUserToolsFile(self, buildData):
             data[self.bStr.gccExePath] = buildData[self.bStr.gccExePath]
             data[self.bStr.gccInludePath] = buildData[self.bStr.gccInludePath]
             data[self.bStr.buildToolsPath] = buildData[self.bStr.buildToolsPath]
-            data[self.bStr.openOCDInterfacePath] = buildData[self.bStr.openOCDInterfacePath]
-            data[self.bStr.openOCDPath] = buildData[self.bStr.openOCDPath]
+            data[self.bStr.pythonPath] = buildData[self.bStr.pythonPath]
+            data[self.bStr.openOcdPath] = buildData[self.bStr.openOcdPath]
+            data[self.bStr.openOcdConfig] = buildData[self.bStr.openOcdConfig]
+            data[self.bStr.stm32SvdPath] = buildData[self.bStr.stm32SvdPath]
+            data[self.bStr.stm32SvdFile] = buildData[self.bStr.stm32SvdFile]

```

Comment:
I agree `openOcdConfig` and `stm32SvdFile` should be removed from here, but think `openOcdPath` and `stm32SvdPath` should remain since they should be common across projects.